packages/vm/tests/api/buildBlock.spec.ts

```diff
@@ -9,12 +9,15 @@ import { VM } from '../../src/vm'
 
 import { setBalance } from './utils'
 
+import type { EEI } from '../../src'
+import type { EVM } from '@ethereumjs/evm'
+
 tape('BlockBuilder', async (t) => {
   t.test('should build a valid block', async (st) => {
     const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Istanbul })
     const genesisBlock = Block.fromBlockData({ header: { gasLimit: 50000 } }, { common })
     const blockchain = await Blockchain.create({ genesisBlock, common, validateConsensus: false })
-    const vm = await VM.create({ common, blockchain })
+    const vm = await VM.create<EVM, EEI>({ common, blockchain })
```

Comment:
Why is this necessary?

Reply:
It's not :facepalm: .  I misread VSCode's red squigglies.  Will revert.